rust/src/lib.rs

```diff
@@ -83,7 +83,7 @@ extern crate suricata_derive;
 pub mod core;
 
 #[macro_use]
-pub(crate) mod debug;
+pub mod debug;
```

Comment:
I think you also need https://github.com/OISF/suricata/pull/12446/commits/8b126705ba0936f5989e0243d679e9ab5141a6b4

Reply:
fixes were merged